Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.17.3
+
+# Install packages
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN apt-get install -y git nodejs netcat
```

Comment:
I don't know if netcat is used here. I think it's mainly for the `whenavail`  script. Similarly, I don't see a use for git anywhere.